src/kv1/read.rs

```diff
@@ -308,6 +308,56 @@ where
     Ok(())
 }
 
+fn make_trip_properties(
+    map_vj_accs: HashMap<String, HashSet<Accessibility>>,
+    collections: &mut Collections,
+) -> Result<()> {
+    let mut map_acc_vjs: BTreeMap<Availability, Vec<String>> = BTreeMap::new();
+    for (vj_id, acc) in map_vj_accs {
+        let avaibility = {
+            if acc.len() == 1 {
+                match acc.iter().next() {
+                    Some(acc) if *acc == Accessibility::Accessible => Availability::Available,
+                    Some(acc) if *acc == Accessibility::NotAccessible => Availability::NotAvailable,
+                    _ => Availability::InformationNotAvailable,
+                }
+            } else {
+                Availability::InformationNotAvailable
+            }
+        };
+        map_acc_vjs
```

Comment:
this map does not seems very useful (and the number of vj can be quite large).

I think you can just `get_or_create` the TripProperty on the fly (maybe by having a local map `Availability` -> `TripPropertyId`)

Reply:
maybe my comment was not very clear, here are some speudo code (not tested at all):
```rust
let mut trip_propeties: BTreeMap<Availability, TripProperty> = BTreeMap::new();
for (vj_id, acc) in map_vj_accs {
        let avaibility = compute_availability();

        let associated_trip_property = trip_properties.entry(availability).or_insert_with(TripProperty {....});
        vj.trip_property_id = Some(associated_trip_property.id);
}
// + add all tripproperties in collections nif not done at the creation
```